rdr_service/participant_enums.py

```diff
@@ -1021,3 +1021,8 @@ class WorkbenchResearcherYesNoPreferNot(messages.Enum):
     YES = 1
     NO = 2
     PREFER_NOT_TO_ANSWER = 3
+
+
+class RemoteIdVerificationStatus(messages.Enum):
+    FALSE = 0
+    TRUE = 1
```

Comment:
Dont think we need this enum class for booleans, we can just use 0/1 in the model attribute and the API will render it correctly.